inspirehep/modules/workflows/tasks/matching.py

```diff
@@ -221,10 +223,9 @@ def article_exists(obj, eng):
 
 def already_harvested(obj, eng):
     """Check if record is already harvested."""
-    if current_app.config.get('PRODUCTION_MODE'):
-        if was_already_harvested(obj.data):
-            obj.log.info('Record is already being harvested on INSPIRE.')
-            return True
+    if was_already_harvested_on_legacy(obj.data):
+        obj.log.info('Record is already being harvested on INSPIRE Legacy.')
```

Comment:
You might want to add here which record is that is already being harvested, though you can ultimately find out by the workflow id, it's easier to debug if you also have the record id or similar.